integrations/extensions/starter-kits/language-model-watsonx/README.md

```diff
@@ -0,0 +1,76 @@
+
+## Upload sample action
+
+Use **Actions Global Settings** to upload the [`watsonx-actions.json`](./watsonx-actions.json) file in this kit to your assistant. For more information, see [Uploading](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import). You may also need to refresh the action Preview chat after uploading in order to get all the session variables initialized before these actions will work correctly.
+
+1. Under "Variables"/"Created by you" (within the Actions page), set the `watsonx_project_id` session variable using the project ID value you obtained [when configuring watsonx above](#configure-watsonx-extension).
+
+**NOTE**: If you import the actions _before_ configuring the extensions, you will see some errors on the actions because it could not find the extensions. Simply configure the extensions (as described [above](#prerequisites)) and re-import the action JSON file.
+
+The starter kit includes [a JSON file with sample actions](./watsonx-actions.json) that are configured to use the watsonx extension.
+
+| Action                        | Description                                                                                                                                                                                   |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Invoke watsonx Generation API | Connects to watson with the selected model and the model input                                                                                                                                |
+| Test model                    | Simple test action that asks what model, length, temperature and prompt you want and then calls "Invoke watsonx Generation API" so the model can generate a response to the specified prompt. |
+
+Note that the "Test model" action includes a step that invokes an extension and includes a parameter named `model_id`. You can set the `model_id` session variable to control which model is used by `Test model`. You can see which models work with the Generate API by viewing the supported foundation models in [the watsonx Prompt Lab](https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/fm-prompt-lab.html?context=wx).
+
+### Session variables
+
+These are the session variables used in this example.
+
+- `model_id`: The id of the watsonx model that you select for this action. Defaults to `google/flan-ul2`.
+- `model_input`: The input to the model.
+- `model_parameters_max_new_tokens` : The maximum number of new tokens to be generated. Defaults to 20.
+- `model_parameters_temperature` : The value used to control the next token probabilities. The range isfrom 0.00 to 1.00; 0.00 makes it deterministic.
```

Comment:
- "isfrom" should have a space
- As I understand it, the minimum temperature should really be 0.05, and that makes it _mostly_ deterministic.